controllers/admin/AdminGroupsController.php

```diff
@@ -367,6 +367,7 @@ public function renderForm()
                         ),
                     ),
                     'hint' => $this->trans('Customers in this group can view prices.', array(), 'Admin.Shopparameters.Help'),
+                    'desc' => $this->trans('Need to hide prices for all groups? Save time, enable catalog mode in Product Settings instead.', array(), 'Admin.Shopparameters.Help'),
```

Comment:
short array syntax :)

Reply:
well all the file is in long array syntax.. it would not be consistent to use only one
And updating the whole file is not exactly in this PR scope